Assets/Resources/Materials/M_GuardProto.mat

```diff
@@ -0,0 +1,83 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
+Material:
```

Comment:
Where did this material come from? This is the same material as on the guard setup branch.